src/gordon_gcp/clients/gdns.py

```diff
@@ -102,10 +102,13 @@ class GDNSClient(http.AIOConnection):
     REVERSE_PREFIX = 'reverse-'
 
     def __init__(self, project=None, auth_client=None, api_version='v1',
-                 session=None):
+                 session=None, default_zone_prefix=None):
         super().__init__(auth_client=auth_client, session=session)
         self.project = project
         self._base_url = f'{self.BASE_URL}/{api_version}/projects/{project}'
+        prefix = f'{default_zone_prefix}-' if default_zone_prefix else ''
```

Comment:
I would leave the - out and have the user put it in the config.  If you really want to leave it, please document that it's added.

Reply:
I thought it was consistent enough with our naming of the zones that it would be unsurprising (an earlier version of this PR actually removed it from the REVERSE_PREFIX) and only added it here as well, since I don't really see that as part of the prefix itself), but I can update the doc.